dhis-2/dhis-services/dhis-service-tracker/src/test/resources/tracker/te_program_with_tea_delete_data.json

```diff
@@ -1,102 +1,12 @@
 {
-  "importMode": "COMMIT",
```

Comment:
I dont think this should be deleted. These tracker integration tests use the TrackerImportParams JSON.

Reply:
Are you talking about the `importMode` or `idSchemes`?
I really want to have the payload as little as possible and not to deal with unnecessary details here

Reply:
I've added them to all our payloads, since that reflects reality AFAIK. I added them in the migration by deserializing and serializing the JSON again (using the adaptations I made to the TrackerImportParams).

Reply:
I put them back.
We can have another look at them later